download_test_vectors.sh

```diff
@@ -9,13 +9,16 @@
 
 set -Eeuo pipefail
 
-VERSIONS=(
-  "v1.6.0-alpha.0"
-)
+# Override versions, eg: TEST_VECTORS_VERSIONS=nightly
+if [[ -n "${TEST_VECTORS_VERSIONS:-}" ]]; then
+	IFS=',' read -ra VERSIONS <<< "$TEST_VECTORS_VERSIONS"
+else
+	VERSIONS=("v1.5.0-beta.5")
```

Comment:
Hm, current version is v1.6.0-alpha.0, why does this set it back to v1.5.0-beta.5?

Reply:
Oops, my mistake! Thanks for catching that. Fixed.